wrangler-docs/functions/date-and-time-functions.md

```diff
@@ -0,0 +1,1010 @@
+set-column result datetime:TimestampOffsetBySeconds(basetimestamp, secondsoffset)
+```
+
+## EpochFromTimestamp
+Returns a UNIX Epoch time value from the given timestamp. If timestamp is null, null will be returned.
+
+#### Namespace
+`datetime`
+
+#### Input
+timestamp(`LocalDateTime`)
+
+#### Output
+epoch time(`Long`)
+
+#### Example
+If `basetimestamp` contains the value `2009–02–13 23:31:30`, then the two following functions are equivalent,
+and return the value `1234567890`
+```
+set-column result datetime:EpochFromTimestamp('2009–02–13 23:31:30')
+set-column result datetime:EpochFromTimestamp(basetimestamp)
+```
+
+## WeekdayFromDate
+Returns the day number of the week from the given date. Base day optionally specifies the day
+that is regarded as the first in the week and is Sunday by default.
+If date is null, null will be returned. If startOfWeek is null, the date's day number of the week will be returned.
+
+#### Namespace
+`datetime`
+
+#### Input
+date(`LocalDate`), [Base day(`String`)]
+
+#### Output
+day(`Integer`)
+
+#### Example
+If `basedate` contains the date `2008-08-18`, then the two following functions are equivalent, and return the value `1`
+```
+set-column result datetime:WeekdayFromDate('2008-08-18')
+set-column result datetime:WeekdayFromDate(basedate)
+```
+If `basedate` contains the date `2008-08-18`, and `baseday` contains `saturday`, then the two following
+functions are equivalent, and return the value `2`
+```
+set-column result datetime:WeekdayFromDate('2008-08-18', "saturday")
+set-column result datetime:WeekdayFromDate(basedate,baseday)
+```
+
+## YeardayFromDate
+Returns the day number in the year from the given date. If date is null, null will be returned.
+
+#### Namespace
+`datetime`
+
+#### Input
+date(`LocalDate`)
+
+#### Output
+day(`Integer`)
+
+#### Example
+If `basedate` contains the date `2008-08-18`, then the two following functions are equivalent,
+and return the value `231`
+```
+set-column result datetime:YeardayFromDate('2008-08-18')
+set-column result datetime:YeardayFromDate(basedate)
+```
+
+## YearFromDate
+Returns the year from the given date. If date is null, null will be returned.
+
+#### Namespace
+`datetime`
+
+#### Input
+date(`LocalDate`)
+
+#### Output
+day(`Integer`)
+
+#### Example
+If `basedate` contains the date `2008-08-18`, then the two following functions are equivalent,
+and return the value `2008`
+```
+set-column result datetime:YearFromDate('2008-08-18')
+set-column result datetime:YearFromDate(basedate)
+```
+
+## YearweekFromDate
+Returns the week number in the year from the given date. If date is null, null will be returned.
+
+#### Namespace
+`datetime`
+
+#### Input
+date(`LocalDate`)
+
+#### Output
+week(`Integer`)
+
+#### Example
+If `basedate` contains the date `2008-08-18`, then the two following functions are equivalent,
+and return the value `33`
+```
+set-column result datetime:YearweekFromDate('2008-08-18')
```

Comment:
these examples all need to be updated since the functions take LocalDate objects and not strings.